cmd/spruce/main_test.go

```diff
@@ -162,7 +162,8 @@ func TestMain(t *testing.T) {
 			stdout = fmt.Sprintf(format, args...)
 		}
 		var stderr string
-		printfStdErr = func(format string, args ...interface{}) {
+		//Edit log stderr function
+		PrintfStdErr = func(format string, args ...interface{}) {
```

Comment:
But this is code scoped to just the `main` package, and isn't referencing the log package.. I guess if importing was an issue, why not just rename `var printfStdErr` to `var PrintfStdErr`, instead of make a new function that did the same thing, wrapped by the original?